File: Documentation Code Snippets/ScanbotSDK Examples/MRZScanner/DetectionOnImage/Objc/MRZDetectionOnImage.m
//
//  MRZDetectionOnImage.m
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 24.03.23.
//

#import "MRZDetectionOnImage.h"
@import ScanbotSDK;

@implementation MRZDetectionOnImage

- (void) detectMRZOnImage {
    
    // Image containing MRZ.
    UIImage *image = [UIImage imageNamed:@"mrzImage"];
    
    // Creates an instance of `SBSDKMachineReadableZoneRecognizer`.
    SBSDKMachineReadableZoneRecognizer *detector = [[SBSDKMachineReadableZoneRecognizer alloc] init];
    
    // Returns the result after running detector on the image.
    SBSDKMachineReadableZoneRecognizerResult *result = [detector recognizePersonalIdentityFromImage:image];
}

@end

File: Documentation Code Snippets/ScanbotSDK Examples/MRZScanner/DetectionOnImage/Swift/MRZDetectionOnImage.swift
//
//  MRZDetectionOnImage.swift
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 24.03.23.
//

import Foundation
import ScanbotSDK

func detectMRZOnImage() {
    
    // Image containing MRZ.
    guard let image = UIImage(named: "mrzImage") else { return }
    
    // Creates an instance of `SBSDKMachineReadableZoneRecognizer`.
    let detector = SBSDKMachineReadableZoneRecognizer()
    
    // Returns the result after running detector on the image.
    let result = detector.recognizePersonalIdentity(from: image)
}
